CHANGELOG.md

```diff
@@ -5,25 +5,39 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+Contributors:
+
+- [@GitHub-Username](Link to GitHub profile) ([#PR-Number](Link to PR))
+- [@bolajiwahab](https://github.com/bolajiwahab) ([#1063])(https://github.com/sqlfluff/sqlfluff/pull/1063)
+- [@silverbullettruck2001](https://github.com/silverbullettruck2001) ([1126](https://github.com/sqlfluff/sqlfluff/pull/1126))
+- [@silverbullettruck2001](https://github.com/silverbullettruck2001) ([1099](https://github.com/sqlfluff/sqlfluff/pull/1099))
```

Comment:
Every contributor should appear once, each PR should be separated by a comma (also don't forget the `#`)